pkg/sync/v15/data/ClusterRole.rbac.authorization.k8s.io/edit.yaml

```diff
@@ -19,6 +19,24 @@ rules:
   - access
   verbs:
   - create
+- apiGroups:
```

Comment:
Do we need these role definitions here if we use aggregations? I think this duplicates the effort

Reply:
this is exactly my question. 
without this sync pod is continuously updating, because there are ALWAYS changes from sync pod perspective.  
https://github.com/openshift/openshift-azure/pull/2129#issuecomment-573478271
https://github.com/openshift/openshift-azure/pull/2129#issuecomment-573901132